backend/requirements/base.txt

```diff
@@ -2,3 +2,4 @@ Django==1.11.3
 wagtail==1.11.1
```

Comment:
'cause the actually default is `sqlite`.

If a collaborator w/o pg start an instances with that commit, the setup will fail.

options:

1. move `psycopg2==2.7.1` in production' dependencies
2. use pg as default in settings_base.py